test/hotspot/gtest/runtime/test_os_windows.cpp

```diff
@@ -703,6 +703,54 @@ TEST_VM(os_windows, reserve_memory_special) {
   TestReserveMemorySpecial_test();
 }
 
+TEST_VM(os_windows, large_page_init_decide_size) {
```

Comment:
should we also have a test that there are multiple page sizes in the list and that they are all multiples of the smallest and do not exceed the maxlargepagesize?